data_diff/diff_tables.py

```diff
@@ -100,11 +102,67 @@ def __iter__(self):
             self.result_list.append(i)
             yield i
 
+        for sign, values in self.result_list:
```

Comment:
Could this replace the behavior of get_stats_string() instead? Or maybe be an additional method like get_stats_tables_string()?

Reply:
I think it can replace `get_stats_string()`. I don't think there's any drawback at all to fully replacing the existing `string_output` since it is not structured (and therefore depended on by some downstream system)--it's just a string.

I will refactor this, and I'm very open to suggestions/recommendations, but I'm going to try to settle on a good design of the output before optimizing the code.

Reply:
@dlawin moved the logic into `get_stats_string` 👍  (still could be optimized from here)